app/services/app_store_update_processor.rb

```diff
@@ -18,10 +19,13 @@ def convert_params(record)
       }
     end
 
-    def update_claim(claim_id, params)
+    def update_claim(claim_id, params, record)
       claim = Claim.find_by(id: claim_id)
 
-      claim&.update!(params)
+      return unless claim
+
+      claim.update!(params)
+      Nsm::AssessmentSyncer.call(claim, record:)
```

Comment:
possible refactor

```ruby
    def update_claim(record)
      claim = Claim.find_by(id: record['application_id'])
      
      return unless claim

      claim.update!(convert_params(record))
      Nsm::AssessmentSyncer.call(claim, record:)
    end
```

and call at line 6

```ruby
  update_claim(record
```

and similar for `update_prior_authority_application` method